.travis.yml

```diff
@@ -31,13 +31,24 @@ matrix:
             - build-essential
             - libgtk-3-dev
             - libxtst-dev
+      before_script:
+        - mkdir -p $HOME/graalvm && cd $HOME/graalvm
+        - wget https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-19.3.0/graalvm-ce-java11-linux-amd64-19.3.0.tar.gz
+        - tar xf graalvm-ce-java11-linux-amd64-19.3.0.tar.gz && cd $TRAVIS_BUILD_DIR
+        - export GRAALVM_HOME=$HOME/graalvm/graalvm-ce-java11-19.3.0/Contents/Home
```

Comment:
Linux does not have the `/Contents/Home` subfolder structure.

Reply:
Yes, I realized...